specification/purview/data-plane/Azure.Analytics.Purview.Share/preview/2022-06-30-preview/share.json

```
@@ -0,0 +1,1796 @@
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "in": "body",
+            "name": "receivedShare",
+            "description": "The updated received share",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ReceivedShare"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted",
+            "schema": {
+              "$ref": "#/definitions/ReceivedShare"
+            },
+            "headers": {
+              "Operation-Location": {
+                "description": "The URL to track the status of the long running operation.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/PurviewShareError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ReceivedShares_CreateOrUpdate": {
+            "$ref": "./examples/ReceivedShares_CreateOrUpdate.json"
+          },
+          "ReceivedShares_UpdateAdlsGen2Account": {
+            "$ref": "./examples/ReceivedShares_UpdateAdlsGen2Account.json"
+          },
+          "ReceivedShares_UpdateBlobAccount": {
+            "$ref": "./examples/ReceivedShares_UpdateBlobAccount.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      },
+      "delete": {
+        "tags": [
+          "ReceivedShare"
+        ],
+        "summary": "Deletes a received share",
+        "description": "Delete a received share",
+        "operationId": "ReceivedShares_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/receivedShareId"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted",
+            "headers": {
+              "Operation-Location": {
+                "description": "The URL to track the status of the long running operation.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Default error response.",
+            "schema": {
+              "$ref": "#/definitions/PurviewShareError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ReceivedShares_Delete": {
+            "$ref": "./examples/ReceivedShares_Delete.json"
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/receivedShares/attached": {
+      "get": {
+        "tags": [
+          "ReceivedShare"
+        ],
+        "summary": "Get a list of attached received shares.",
+        "description": "List attached received shares",
+        "operationId": "ReceivedShares_GetAllAttached",
```

The verb in the operationId for `get` on a collection endpoint should be "list".
```suggestion
        "operationId": "ReceivedShares_ListAttached",
```

---

Done.